app/view/grid/UserListModel.js

```diff
@@ -3,7 +3,7 @@ Ext.define('MoMo.admin.view.grid.UserListModel', {
     alias: 'viewmodel.momo-userlist',
 
     data: {
-        title: 'All users',
+        title: 'All sers',
```

Comment:
The data will be overridden by the values from the i18n files anyway, this was just a leftover from a test (fixed it anyway).